### PR TITLE
Add post about SR2022 competition date

### PR DESCRIPTION
**Pull request by RealOrangeOne**

### CODE REVIEW

File: _events/sr2022/competition.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+## Schedule
+
+The competition will be held from *Saturday 23rd April* until *Sunday 24th April*. Doors open 9am and we aim to finish around 6pm.
+
+## Location
+
+The competition will be held in the Students' Union building at the [University of Southampton's Highfield campus][soton-campus-directions]:
+
+Student's Union
```
Comment (antoinepetty) [inline _events/sr2022/competition.md:19]:
```suggestion
Students' Union
```
```suggestion
Student's Union
```

File: _events/sr2022/competition.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+University of Southampton
+University Road
+Southampton
+SO17 1BJ
+United Kingdom
```
Comment (sedders123) [inline _events/sr2022/competition.md:24]: These newlines get swallowed unless we add these breaks.
```suggestion
Student's Union<br>
University of Southampton<br>
University Road<br>
Southampton<br>
SO17 1BJ<br>
United Kingdom<br>
```

File: _events/sr2022/competition.md (hunk 1 above)
Comment (antoinepetty) [reply _events/sr2022/competition.md:19]: I have no idea why this suggested twice

File: _events/sr2022/competition.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+## Parking
+
+We have a limited amount of free parking on site for competitors.
+
+![parking information]({{ '/resources/sr2022/parking-map.png' | prepend: site.baseurl }})
+
+## COVID-19
+
+Please see our [covid updates]({{ '/covid-19' | prepend: site.baseurl }}) page.
```
Comment (sedders123) [inline _events/sr2022/competition.md:36]: Nit: We seem to have always referred to Covid with its full name would be nice to be consistent here.
```suggestion
Please see our [COVID-19 updates]({{ '/covid-19' | prepend: site.baseurl }}) page.
```

**Review by sedders123 — COMMENTED** [review]
Looks good! Just a couple of small suggestions

**Review by sedders123 — APPROVED** [review]

**Pull request merged**